Sources/Overlays/_Testing_Foundation/Attachments/Attachment+URL.swift

```diff
@@ -0,0 +1,161 @@
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+//
+
+#if canImport(Foundation)
+@_spi(Experimental) @_spi(ForSwiftTestingOnly) public import Testing
+public import Foundation
+
+#if SWT_TARGET_OS_APPLE && canImport(UniformTypeIdentifiers)
```

Comment:
Is `#if SWT_TARGET_OS_APPLE` alone sufficient? Although we do need runtime availability checks, the `UniformTypeIdentifiers` module should always be available at build-time on Apple platforms, right?

```suggestion
#if SWT_TARGET_OS_APPLE
```

Reply:
No comment. :)